beetmoverscript/script.py

```diff
@@ -313,6 +316,16 @@ def list_bucket_objects(context, s3_resource, prefix):
                     upload_to_s3(context=context, s3_key=destination, path=source)
                 )
             )
+
+            if is_partner_public_task(context):
+                # we trim the full destination to the part after
+                # candidates/61.0b2-candidates/buildN/
```

Comment:
Nit: let's make this comment generic like `candidates/{version}-candidates/buildN/` for the ease of reading.